docs/plugin-transform-unicode-escapes.md

```diff
@@ -1,21 +1,19 @@
 ---
 id: babel-plugin-transform-unicode-escapes
 title: @babel/plugin-transform-unicode-escapes
-sidebar_label: transform-unicode-escapes
+sidebar_label: unicode-escapes
 ---
 
-Compile ES2015 Unicode escapes to ES5
```

Comment:
Why was this deleted?

Reply:
Didn't seem that helpful as a description but can add it back, we should add more to all of these maybe

Reply:
Yeah the name and example are probably enough